scripts/package.js

```diff
@@ -0,0 +1,21 @@
+require('bluebird-global')
```

Comment:
PP: Convert this into a Typescript file and call it using ts-node.

Reply:
I plan to do this, i'm just looking for the right setup so we can use the same in the main repository. I'll open another PR when I get this right